pkg/daemon/update.go

```diff
@@ -305,9 +305,10 @@ func Reconcilable(oldConfig, newConfig *mcfgv1.MachineConfig) (*MachineConfigDif
 
 	// Ignition section
 	// First check if this is a generally valid Ignition Config
-	rpt := validate.ValidateWithoutSource(reflect.ValueOf(newIgn))
-	if rpt.IsFatal() {
-		return nil, errors.Errorf("invalid Ignition config found: %v", rpt)
+
+	ignErrMsg := ctrlcommon.ValidateIgnition(newIgn)
+	if ignErrMsg != nil {
+		return nil, ignErrMsg
 	}
```

Comment:
same as https://github.com/openshift/machine-config-operator/pull/894/files#r300422961 - no need for the extra err variable

Reply:
Done.